targets/TARGET_STM/TARGET_STM32F7/TARGET_STM32F767xI/TARGET_UHURU_RAVEN/system_clock.c

```diff
@@ -0,0 +1,262 @@
+/* mbed Microcontroller Library
+* Copyright (c) 2006-2019 ARM Limited
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+  * This file configures the system clock as follows:
+  *--------------------------------------------------------------------
+  * System clock source   | 1- USE_PLL_HSE_EXTC (external 8 MHz clock)
+  *                       | 2- USE_PLL_HSE_XTAL (external 8 MHz xtal)
+  *                       | 3- USE_PLL_HSI (internal 16 MHz clock)
+  *--------------------------------------------------------------------
+  * SYSCLK(MHz)           |            216
+  * AHBCLK (MHz)          |            216
+  * APB1CLK (MHz)         |             54
+  * APB2CLK (MHz)         |            108
+  * USB capable (48 MHz)  |            YES
+  *--------------------------------------------------------------------
+**/
+
+#include "stm32f7xx.h"
+#include "nvic_addr.h"
+#include "mbed_error.h"
+#if TARGET_UHURU_RAVEN
```

Comment:
I might be missing something, but why is this needed if this file is inside of its own TARGET subdir?